### DISCUSSION TITLE
FreeBSD

### DISCUSSION THREAD

**Discussion by probonopd** [FZBunny/applepi#5] [General]
Just wanted to let you know that this runs nicely on FreeBSD ([helloSystem](https://hellosystem.github.io/)).

```
git clone https://github.com/FZBunny/applepi/
cd applepi
qmake-qt5 .
gmake -j8
./bin/applepi
```

![image](https://github.com/FZBunny/applepi/assets/2480569/286c928a-b913-4b76-8b31-525614b23a32)

Now, how to get into BASIC...?

**Comment by probonopd** [comment]
https://prodos8.com/

![image](https://github.com/FZBunny/applepi/assets/2480569/33932bcc-70de-4fec-93a3-f5f180aa900c)

**Comment by FZBunny** [comment]
Why, thank you!  (Of course, all credit must go to the Qt developers.  A
wonderful library.)
